### DISCUSSION TITLE
building-latex-projects-on-windows-easily-with-docker/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#445] [Announcements]
# building-latex-projects-on-windows-easily-with-docker/

In this post I describe how I build LaTeX projects on a Windows machine by using Docker and the blang/latex image

https://andrewlock.net/building-latex-projects-on-windows-easily-with-docker/

**Comment by jvmlet** [comment]
Andrew,  being a huge fan of your blogs, I couldn't hold myself to NOT react to this one.
Have you heard about Antora ? 
Built on top of asciidoc, it can render beautiful web site ([showcases](https://gitlab.com/antora/antora.org/-/issues/20)), and the fact that  you are writing asciidoc opens an endless opportunities for backend renderers (pdf,epub ect), including integration with Kroki (for diagrams rendering ) and much more.....
All the toolkits are  included (docker,cli, extensions, plugins, you name it)
